compiler/exe4j/compiled-with-exe4j.yml

```diff
@@ -0,0 +1,26 @@
+rule:
+  meta:
+    name: compiled with exe4j
+    namespace: compiler/exe4j
+    author: johnk3r
+    scope: file
+    att&ck:
+      - Defense Evasion::Obfuscated Files or Information::Compile After Delivery [T1027.004]
+    examples:
+      - 6b25f1e754ef486bbb28a66d46bababe:0x404EDE
+  features:
+    - or:
+      - string: exe4j_log
+      - string: install4j_log
+      - string: exe4j_java_home
+      - string: install4j
+      - string: exe4j.isinstall4j
+      - string: /com/exe4j/runtime/exe4jcontroller/i
+      - string: /com/exe4j/runtime/winlauncher/i
+      - string: EXE4J_LOG
+      - string: INSTALL4J_LOG
+      - string: EXE4J_JAVA_HOME
+      - string: INSTALL4J
+      - string: EXE4J.ISINSTALL4J
+      - string: /COM/EXE4J/RUNTIME/EXE4JCONTROLLER/i
+      - string: /COM/EXE4J/RUNTIME/WINLAUNCHER/i
```

Comment:
These are covered by the two regexes above. The `i` suffix indicates case-insensitive matching. Please remove the duplicates and then we can merge. Thanks for your great work here!

Reply:
```suggestion
```